src/main/java/mods/railcraft/common/blocks/multi/TankDefinition.java

```diff
@@ -61,7 +61,7 @@ public void defineRecipes(Block block) {
             Crafters.blastFurnace().newRecipe(block)
                     .name("railcraft:smelt_iron_tank")
                     .time(IBlastFurnaceCrafter.SMELT_TIME * 4)
-                    .output(RailcraftItems.INGOT.getStack(4, Metal.STEEL)).slagOutput(4)
+                    .output(RailcraftItems.NUGGET.getStack(4, Metal.STEEL)).slagOutput(1)
```

Comment:
Why? Didn't I just revert this change? Almost all the other smelting recipes are 1:1 on the iron:steel ratio.

Reply:
this is for #1718. Four iron plates can craft 8 tanks walls.

Reply:
Gotcha!